src/solvers/dgsem_tree/indicators_1d.jl

```diff
@@ -89,26 +89,58 @@ function (indicator_hg::IndicatorHennemannGassner)(u::AbstractArray{<:Any,3},
   end
 
   if (alpha_smooth)
-    # Diffuse alpha values by setting each alpha to at least 50% of neighboring elements' alpha
-    # Copy alpha values such that smoothing is indpedenent of the element access order
-    alpha_tmp .= alpha
-
-    # Loop over interfaces
-    for interface in eachinterface(dg, cache)
-      # Get neighboring element ids
-      left  = cache.interfaces.neighbor_ids[1, interface]
-      right = cache.interfaces.neighbor_ids[2, interface]
-
-      # Apply smoothing
-      alpha[left]  = max(alpha_tmp[left],  0.5 * alpha_tmp[right], alpha[left])
-      alpha[right] = max(alpha_tmp[right], 0.5 * alpha_tmp[left],  alpha[right])
-    end
+  
+    apply_smoothing!(u, mesh, equations, dg, alpha, alpha_tmp, cache)
+
   end
 
   return alpha
 end
 
 
+function apply_smoothing!(u::AbstractArray{<:Any, 3}, mesh::TreeMesh{1}, equations, dg::DGSEM, alpha, alpha_tmp, cache)
+
```

Comment:
For consistency, please do not add empty lines at the beginning or end of blocks, ie, functions, begin...end, if etc.